src/qutipinterface.jl

```diff
@@ -0,0 +1,184 @@
+# TODO : QuBase.jl complex WIP. Issue 38. https://github.com/JuliaQuantum/QuBase.jl/pull/38
+Base.complex{B<:QuBase.OrthonormalBasis}(qarr::QuBase.AbstractQuArray{B}) = QuBase.similar_type(qarr)(complex(coeffs(qarr)), bases(qarr))
+
+@doc """
+Inspired by `sesolve`[http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.sesolve] of QuTiP in QuDynamics
+
+`sesolve` in QuDynamics, extended version.
+
+### Arguments
+
+Inputs :
+* H/SE/LVNE :: AbstractQuArray/QuSchrodingerEq/QuLiouvillevonNeumannEq Types
+
+  This parameter supports Hamiltonian of the system, Schrodinger Equation, LiouvillevonNeumann Equation
+* psi :: AbstractQuArray
+
+  Initial state vector or density matrix of the system
+* tlist
+
+  Time step array/range
+* e_ops
+
+  Operators whose expectation values are to computed in the evolved states
+* solver
+
+  Keyword for selecting the solver from the various solvers supported by QuDynamics
+
+Output :
+* Array of tuples, which specify the time-step and the corresponding evolved state
+""" ->
+function sesolve(H::QuBase.AbstractQuArray, psi::QuBase.AbstractQuArray, tlist; solver=QuExpmV())
+    time = collect(tlist)[2:end]
+    evolved_state = Array(QuBase.AbstractQuArray, length(tlist)-1)
+    i = 1
+    for (t, psi) in QuStateEvolution(H, psi, tlist, solver)
+        evolved_state[i] = psi
+        i = i + 1
+    end
+    return time, evolved_state
+end
+
+sesolve(SE::QuSchrodingerEq, psi::QuBase.AbstractQuVector, tlist; solver=QuExpmV()) = sesolve(SE.hamiltonian, psi, tlist; solver=solver)
+sesolve(LVNE::QuLiouvillevonNeumannEq, rho0::QuBase.AbstractQuMatrix, tlist; solver=QuExpmV()) =  sesolve(LVNE.liouvillian, rho0, tlist; solver=solver)
+
+function sesolve(H::QuBase.AbstractQuArray, psi::QuBase.AbstractQuVector, tlist, eops::Vector; solver=QuExpmV())
```

Comment:
It is much more performant and memory friendly to do the same loop as above and storing only the expectation values. Note that you can use an ordinary matrix to store the expectation values, maybe with `Complex128` elements.